doc/rados/configuration/common.rst

```diff
@@ -100,75 +104,78 @@ Additionally, you should enable message signing. See `Cephx Config Reference`_ f
 OSDs
 ====
 
-Ceph production clusters typically deploy :term:`Ceph OSD Daemons` where one node
-has one OSD daemon running a Filestore on one storage device. The BlueStore back
-end is now default, but when using Filestore you specify a journal size. For example:
+When Ceph production clusters deploy :term:`Ceph OSD Daemons`, the typical
+arrangement is that one node has one OSD daemon running Filestore on one
```

Comment:
one OSD daemon for each storage device.  Probably time to remove the Filestore bit below too, though that should probably be left in for earlier releases.

Reply:
unfinished business